packages/scandipwa/src/component/Notification/Notification.component.js

```diff
@@ -110,7 +110,14 @@ export class Notification extends PureComponent {
 
         return (
             <div block="Notification" mods={ mods } ref={ this.notification } id={ id }>
-                <button block="Notification" elem="Button" onClick={ this.hideNotification }>Close</button>
+                <button
+                  block="Notification"
+                  elem="Button"
+                // eslint-disable-next-line react/jsx-no-bind
+                  onClick={ this.hideNotification.bind(this) }
```

Comment:
You can bind a method like this:
```javascript
hideNotification = this.hideNotification.bind(this);
```
For more examples see:
* https://github.com/scandipwa/scandipwa/blob/master/packages/scandipwa/src/component/CarouselScroll/CarouselScroll.component.js#L52-L54
* https://github.com/scandipwa/scandipwa/blob/master/packages/scandipwa/src/component/CartCoupon/CartCoupon.component.js#L42-L48

Reply:
Thank you, corrected!